website/benutzerhandbuch.md

```diff
@@ -0,0 +1,336 @@
+---
```

Comment:
I'm not very familiar with the whole jekyl structure. But why are files `modelbaker.md`, `benutzerhandbuch.md` und `technischeumsetzung.md` in the main folder and not in nested folder like e.g. `_services/umsetzung/qgismodelbaker/` etc.?